spring/src/main/java/org/axonframework/spring/authorization/AuthorizationMessageDispatchInterceptor.java

```diff
@@ -0,0 +1,41 @@
+package org.axonframework.spring.authorization;
+
+import org.axonframework.commandhandling.CommandMessage;
+import org.axonframework.messaging.MessageDispatchInterceptor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiFunction;
+
+/**
+ * Message dispatch interceptor that adds the {$code username} and {$code authorities} from the authorized principle
+ *
+ * @author Roald Bankras
```

Comment:
Nit: please add `@since 4.10.0` here.

```suggestion
 * Message dispatch interceptor that adds the {$code username} and {$code authorities} from the authorized principle.
 *
 * @author Roald Bankras
 * @since 4.10.0
```